### ISSUE TITLE
Stuck when --tid option is given more than twice

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#48]
`--tid` option is to show only specified thread output but if `--tid` option is used twice, uftrace stucks and does not finish.
```
$ uftrace replay --tid 17490 -D 1
# DURATION    TID     FUNCTION
            [17490] | base::_GLOBAL__N_1::ThreadFunc() {

uftrace stopped tracing with remaining functions
================================================
task: 17490
[5] base::MessagePumpDefault::Run
[4] base::MessageLoop::RunHandler
[3] base::RunLoop::Run
[2] base::Thread::Run
[1] base::Thread::ThreadMain
[0] base::_GLOBAL__N_1::ThreadFunc

$ uftrace replay --tid 17471 -D 1
# DURATION    TID     FUNCTION
  16.943 ms [17471] | _GLOBAL__N_1::SpinLock_InitHelper::SpinLock_InitHelper();
  26.989 ms [17471] | LoggerRegisterHelperChildProcessMsg_Shutdown::LoggerRegisterHelperChildProcessMsg_Shutdown();
  60.363 us [17471] | LoggerRegisterHelperChildProcessMsg_GetChildNonPersistentHistogramData::LoggerRegisterHelperChildProcessMsg_GetChildNonPersistentHistogramData();
  32.766 us [17471] | LoggerRegisterHelperAccessibilityMsg_SetFocus::LoggerRegisterHelperAccessibilityMsg_SetFocus();
  58.275 us [17471] | LoggerRegisterHelperAccessibilityHostMsg_LocationChanges::LoggerRegisterHelperAccessibilityHostMsg_LocationChanges();
    ...
  56.909 us [17471] | SkRTConf::SkRTConf();
  56.835  s [17471] | main();
  74.205 us [17471] | SkRTConfRegistry::~SkRTConfRegistry();
   3.217 ms [17471] | GoogleInitializer::~GoogleInitializer();
```
Here is the problematic example:
```
$ uftrace replay --tid 17490 --tid 17471 -D 1
  ( ... does not finish ... )
```
I know that the correct usage of --tid is as follows:
```
--tid=TID[,TID,...]
      Only print functions called by the given threads.  To see the list of threads in the data file, you can use uftrace report --threads or uftrace info.
```
But it's better to prevent using `--tid` option is used more than twice.

**Comment by namhyung** [comment]
Oops, it's a bug not updated when I changed it to use `opt_add_string()` which separates arguments using ';' (instead of ':').  Please see `setup_task_filter()`.  Also I think it'd be better supporting `--tid` option to be used more than once.  Is it possible for you to send a PR with a document change?

**Comment by honggyukim** [comment]
It works fine now even with the following options:
```
$ uftrace replay --tid 17490 --tid 17471,17496 -D 1
```
I will send a PR soon. Thanks for your help!

**Comment by honggyukim** [comment]
I've just sent the PR #49. Please have a look at it. Thanks!

**Comment by namhyung** [comment]
Fixed by #49
